infrastructure/internals/terraform/terraform.go

```diff
@@ -112,3 +117,43 @@ func RunInfrastructureChanges(workspaceDir string, workspaceName string) (string
 
 	return out, err
 }
+
+// DestroyInfrastructure destroy existing infrastructure
+func DestroyInfrastructure(workspaceDir string) error {
+	fmt.Println("Distroying Infrastructure...")
+	s := spinner.New(spinner.CharSets[11], 100*time.Millisecond) // Build our new spinner
```

Comment:
Do we need to create a new spinner? 
There's a spinner in the utility as well

Reply:
I have tried but unable to import due to other spinner module that is already imported but now i will modify it and use it